t/lib/TestDB.pm

```diff
@@ -12,11 +12,37 @@ sub new {
 
     open my $fh, '<', 'share/test_data.sql';
 
+    my $continued_line = "";
+    my $last_line_was_continued = 0;
```

Comment:
Do you really need `$last_line_was_continued` or is it enough to check if `continued_line` is non-empty?